package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "hof-frontend-assets",
+  "version": "0.1.0",
+  "main": "./bin/index.js",
+  "description": "Frontend JS, CSS and images for a form service",
+  "license": "GPLv3",
+  "author": "UKHomeOfficeForms",
+  "repository": "https://github.com/UKHomeOfficeForms/hof-frontend-assets.git",
+  "bugs": "https://github.com/UKHomeOfficeForms/hof-frontend-assets/issues",
+  "bin": {
+    "hof-frontend-assets": "./bin/index.js"
+  },
+  "scripts": {
+    "test": "npm-run-all lint style browser:test unit:coverage",
+    "browser:test": "karma start --single-run",
+    "unit:test": "mocha test/unit",
+    "cover": "istanbul cover _mocha -- test/unit",
+    "check-coverage": "istanbul check-coverage",
+    "unit:coverage": "npm-run-all cover check-coverage",
+    "snyk": "snyk test",
+    "snyk:dev": "snyk test --dev",
+    "lint": "eslint .",
```

Comment:
@joefitter can you remind me why we enforce npm 3? I know it's not so we can bypass explicitly adding deps. However, in light of the fact that we are using npm 3, it seemed superfluous to add deps that are already included.